client/src/href-builder.js

```diff
@@ -52,7 +53,11 @@ module.exports = function(config){
 
     prepareServerGet: function(baseUrl, component, options) {
       var urlPath = component.name + (component.version ? '/' + component.version : '');
-      var qs = options.parameters ? ('/?' + querystring.stringify(options.parameters)) : '';
+
+      var qs = '';
+      if (component.parameters || options.parameters) {
+        qs = '/?' + querystring.stringify(mergeObjects(component.parameters, options.parameters));
```

Comment:
what about using _.extend. We use it a lot everywhere so it would be more consistent.
```js
var _ = require('underscore')
var p = _.extend(options.parameters, component.parameters)
// example
_.extend({b:3, c:4}, {a:1, b:2})
=> { b: 2, c: 4, a: 1 }
```